src/Components/Web/src/Forms/InputBase.cs

```diff
@@ -50,6 +50,11 @@ public abstract class InputBase<TValue> : ComponentBase, IDisposable
         /// </summary>
         [Parameter] public Expression<Func<TValue>>? ValueExpression { get; set; }
 
+        /// <summary>
+        /// Gets or sets the display name for this field.
+        /// </summary>
+        [Parameter] public string? DisplayName { get; set; }
```

Comment:
You'll need to regen the ref assemblies. See https://github.com/dotnet/aspnetcore/blob/master/docs/ReferenceAssemblies.md#when-changing-public-api

Reply:
I'm having an issue with the command, I'm getting:
> A compatible installed .NET Core SDK for global.json version [5.0.100-preview.7.20330.3] from [c:\aspnetcore\global.json] was not found

Am I doing something wrong? I'm executing `dotnet msbuild /t:GenerateReferenceSource` inside `src/Components/Web/src`.

Reply:
My bad, didn't run the `activate` script!